src/functions.js

```diff
@@ -144,5 +243,6 @@ module.exports = {
   average,
   paintShop,
   sales,
-  secondLargest
+  secondLargest,
+  factorial
 };
```

Comment:
General comments:

1. A discussion on code style would be a good idea for all students, as things like white space, nesting and variable naming would benefit everyone.
2. It's a good idea to remove any unused or commented out sections before submitting a PR, as it aids readability for a reviewer.